jazz-build-module/env-loader.groovy

```diff
@@ -0,0 +1,158 @@
+#!groovy?
+import groovy.json.JsonSlurperClassic
+import groovy.json.JsonOutput
+import groovy.transform.Field
+
+/*
+* Logic for accessing certain values from the given jazz-installer-vars json
+*/
+
+@Field def g_aws_credential_id
+@Field def g_env_name_prefix
+@Field def g_cloudfront_origin_id
+@Field def g_cognito
+@Field def g_repository
+@Field def g_lambda_build
+@Field def g_website_build
+@Field def g_api
+@Field def g_jazz
+@Field def g_jazz_s3
+@Field def g_website_s3
+@Field def g_jenkins
+@Field def g_bitbucket
+
+echo "the module, 'env-loader', loaded successfully... congratulations..."
+
+def initialize(installerVars){
+  if(installerVars != null){
+		def resultJson = parseJson(installerVars)
+
+    echo "assigning env values from installer..."
+
+    setCognito(resultJson.COGNITO)
```

Comment:
While I appreciate all the methods to map keys to methods, I believe this will cause a maintenance overhead with no additional value. 

Let this module return the complete JSON object which will be referenced in build packs as configLoader.AWS_CREDENTIAL_ID,  configLoader.API.API_ID_PROD and so on ...